src/routes/graph/add-pin.ts

```diff
@@ -23,11 +25,13 @@ interface EffectiveRequest {
     fields: FieldsToServerFull;
     typeKey: string | null;
     edgeHashId: string | null;
+    pinGridsHashIds: string[];
```

Comment:
Should this not read | null if you add null as the default value for pinGridHashIds?

Reply:
Yes, I think it should too, at least to match the Joi definition. I'll open a PR

Reply:
https://github.com/withthegrid/platform-sdk/pull/460